carbon-design-system/carbon
# build(react): add a script that scaffolds out RTL tests

REF #10184 

Since we've been updating tests to use RTL, I noticed that just scaffolding out the tests for all the props was taking a while and I thought that creating the test file for the new RTL tests could be automated. 

If it works (lol) this script should prompt you to choose a component to write tests for and then create a test file for that component with tests in place for its props and the appropriate imports. It's definitely pretty basic since I wanted it to work for every component so it's more of a jumping off point for tests than anything else. 

Since I like to refer to the other tests for help sometimes, this creates a new test file instead of overriding the previous one. 

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

#### Testing / Reviewing

1. Pull down the PR
2.  Run `yarn` to install `enquirer`
3.  In `packages/react`, run `node tasks/build-test-rtl.js` and follow the prompt. 
4. After selecting a component, verify that a new test file has been made with the name `${component}-test__copy.js`
